codility_training/lessons.lesson07.StacksandQueues.Fish/sAp00n.py

```diff
@@ -23,10 +23,11 @@ def fish_eat_func(fish_stack, Bi, Ai):
     if len(fish_stack) == 0:
         fish_stack.append((Bi, Ai))
         return
-    if fish_stack[-1][0] != Bi and Ai > fish_stack[-1][-1]:
+    while fish_stack[-1][0] != Bi and Ai > fish_stack[-1][-1]:
         fish_stack.pop()
```

Comment:
```suggestion
    b, a = fish_stack[-1]
    while b != Bi and Ai > a:
        b, a = fish_stack.pop()
```
배열의 경우 탐색시간이 소요되기 때문에 이런식으로 변수에 할당하는것도 좋은 방법일 것 같습니다~